docs/LwM2M_Client/Nordic/Runtime_certificate.md

```diff
@@ -0,0 +1,108 @@
+# Runtime Certificate and Private Key Configuration
+
+## Introduction
+Anjay library supports secure connection between device and server. Encryption can be achieved with the use of Pre-Shared Key as it was described in [nRF9160](nRF9160.md) and [Thingy:91](Thingy91.md) pages or with certificate-based cryptography.
+
+In **Certificate Mode** an asymmetrical algorithm is used to authenticate the connection endpoints and initialize payload encryption. Appropriate certificates need to be generated for both the LwM2M Client and the LwM2M Server. Public certificates of both parties are mutually available, and each party also has access to its corresponding private key.
+
+You can read more about secure communication on [Anjay's documentation](https://avsystem.github.io/Anjay-doc/BasicClient/BC-Security.html) page.
+
+## Prerequisites
+* The nRF9160DK board with a USB cable.
+* Cloned [Anjay-zephyr-client](https://github.com/AVSystem/Anjay-zephyr-client#getting-started) repository
+* Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
+* Installed [OpenSSL](https://www.openssl.org/source/)
+* Installed **minicom** (for Linux), **RealTerm**, **PuTTy** (for Windows), or another serial communication program.
+* An active [Coiote IoT DM](https://eu.iot.avsystem.cloud/) user account.
+
+!!! Note
+    In this tutorial we will use the nRF9160DK board as an example.
+
+## Build and flash the device
+0. Connect the nRF9160DK board to a USB port of your machine.
+0. Go to your local **Anjay-zephyr-client** directory
+0. Set West manifest path to `Anjay-zephyr-client/demo`, manifest file to `west-nrf.yml`, and do `west update`:
+
+    ```
+    west config manifest.path Anjay-zephyr-client/demo
```

Comment:
If we just applied
`Go to your local **Anjay-zephyr-client** directory`
now the correct path would be just `demo`. Since in the next step we navigate to `demo` directory anyway, maybe go there in the first place and skip `west config manifest.path`?

Reply:
bump